### ISSUE TITLE
Overview page: Markdown issues

### ISSUE THREAD

**Issue by MakisH** [solids4foam/solids4foam.github.io#63]
In the [Overview](https://www.solids4foam.com/documentation/overview.html) page ([this file](https://github.com/solids4foam/solids4foam.github.io/blob/master/documentation/overview.md)), there are a few rendering issues, which I cannot understand where they originate from (notice the URLs):

![Screenshot from 2024-12-12 19-23-00](https://github.com/user-attachments/assets/d83f907e-134d-4a61-a544-024494313fea)

I suggest checking the whole documentation with a tool such as https://github.com/DavidAnson/markdownlint.

This is in the context of my review for the JOSS publication (https://github.com/openjournals/joss-reviews/issues/7407).

**Comment by philipcardiff** [comment]
Thanks, @MakisH, for all the excellent contributions.

I need to look into this more. The rendering is fine for me on macOS using Safari and Chrome (attached); let me check other systems.

The `markdownlint` is very nice; I will add it.

<img width="1663" alt="image" src="https://github.com/user-attachments/assets/3e620014-5d5b-4046-84a2-b93c758263c3" />

**Comment by MakisH** [comment]
@philipcardiff this was already fixed in #64, which is now merged.

**Comment by philipcardiff** [comment]
> @philipcardiff this was already fixed in #64, which is now merged.

Ah, I see. I was reading the issues/pull-requests out of order!

**Comment by philipcardiff** [comment]
FYI, I added the Markdown Lint action; it is currently set to be run manually. It highlights issues which I will address.

**Comment by philipcardiff** [comment]
The Markdown Lint action has now been added to the solids4foam and solids4foam.github.io repositories, and runs automatically on pushes and pull requests to the `development` and `master` branches.

After lots of editting (thanks ChatGPT), the checks pass on both repositories. This is a nice feature to have.

**Comment by MakisH** [comment]
Great! Just a heads-up, from experience: it might eventually get a bit annoying, and you might need to add a few rule exceptions. But overall, yes, it is something very nice to have.

You probably already experienced a similar situation with the link check, [I see](https://github.com/solids4foam/solids4foam.github.io/blob/master/.markdown_linkcheck_config.json).

**Comment by philipcardiff** [comment]
Yep, for some reason, the link check fails with some valid DOI links.

For now, I explicitly tell Markdown-lint which [directories to check](https://github.com/solids4foam/solids4foam.github.io/blob/master/.github/workflows/markdownlint.yml), which seems to work fine. I expect that I have introduced some minor rendering issues as I made the md files compliant, but we can fix these as we find them.
